webapi/src/main/scala/org/knora/webapi/Main.scala

```diff
@@ -76,7 +76,7 @@ object Main extends App with LiveCore with KnoraService {
             """.stripMargin)
     } else {
         /* Start the HTTP layer, allowing access */
-        startService(true)
+        startService(false)
```

Comment:
Could you add a comment why this needs to be false?

Reply:
done